backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/dataImport/importInvoiceCandidate.feature

```diff
@@ -0,0 +1,170 @@
+      | pl_1       | ps_1                          | DE                        | EUR                 | pl_280822 | true  | false         | 2              | true         |
+    And metasfresh contains M_PriceList_Versions
+      | Identifier | M_PriceList_ID.Identifier | ValidFrom  |
+      | plv_1      | pl_1                      | 2022-08-01 |
+    And add HTTP header
+      | Key          | Value      |
+      | Content-Type | text/plain |
+      | Accept       | */*        |
+
+  @from:cucumber
+  Scenario: Import sales I_Invoice_Candidate from csv - setting all available fields
+    Given metasfresh initially has no I_Invoice_Candidate data
+    And metasfresh contains M_Products:
+      | Identifier | Name                 | Value                      |
+      | product_1  | Product_25_08_2022_1 | Product_Value_25_08_2022_1 |
+    And metasfresh contains M_ProductPrices
+      | Identifier | M_PriceList_Version_ID.Identifier | M_Product_ID.Identifier | PriceStd | C_UOM_ID.X12DE355 | C_TaxCategory_ID.InternalName |
+      | pp_1       | plv_1                             | product_1               | 10.0     | PCE               | Normal                        |
+    And metasfresh contains C_BPartners:
+      | Identifier     | Name                      | Value                         | OPT.IsVendor | OPT.IsCustomer | M_PricingSystem_ID.Identifier | OPT.InvoiceRule |
+      | billBpartner_1 | BillBPartner_25_08_2022_1 | BillBPartner_Value_25_08_2022 | N            | Y              | ps_1                          | I               |
+    And metasfresh contains C_BPartner_Locations:
+      | Identifier       | GLN           | C_BPartner_ID.Identifier | OPT.IsShipTo | OPT.IsBillTo |
+      | billBPLocation_1 | 1239977890123 | billBpartner_1           | true         | true         |
+    And metasfresh contains AD_Users:
+      | AD_User_ID.Identifier | Name                  | OPT.C_BPartner_ID.Identifier | OPT.C_BPartner_Location_ID.Identifier |
+      | billBPUser_1          | BillBPartnerContact_1 | billBpartner_1               | billBPLocation_1                      |
+    And store DataImport string requestBody in context
+      | Bill_BPartner_ID.Identifier | Bill_Location_ID.Identifier | Bill_User_ID.Identifier | M_Product_ID.Identifier | OPT.DateOrdered | QtyOrdered | OPT.QtyDelivered | OPT.X12DE355 | IsSOTrx | OPT.DocBaseType | OPT.DocSubType | OPT.PresetDateInvoiced | OPT.Description | OPT.POReference | OPT.InvoiceRule |
+      | billBpartner_1              | billBPLocation_1            | billBPUser_1            | product_1               | 2022-08-25      | 5          | 3                | PCE          | true    | ARI             | LS             | 2022-08-26             | DescriptionTest | PORef           | D               |
+
+    When the metasfresh REST-API endpoint path 'api/v2/import/text?dataImportConfig=InvoiceCandidate' receives a 'POST' request with the payload from context and responds with '200' status code
```

Comment:
wouldn't it be better to call this EP with runSynchronous=true to avoid the FUD of possible flackyness on slow CI-servers?

Reply:
fully agree, working on it